connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/UplynkListener.kt

```diff
@@ -0,0 +1,10 @@
+package com.theoplayer.android.connector.uplynk
+
+import com.theoplayer.android.connector.uplynk.network.AssetInfoResponse
+import com.theoplayer.android.connector.uplynk.network.PreplayResponse
+
+interface UplynkListener {
+    fun onPreplayResponse(response: PreplayResponse)
+    fun onAssetInfoResponse(response: AssetInfoResponse)
```

Comment:
Give all methods an empty default implementation. This makes it easier for customers to listen for only *one* event, and it also makes it easier for us to add additional events in the future.
```suggestion
    fun onPreplayResponse(response: PreplayResponse) {}
    fun onAssetInfoResponse(response: AssetInfoResponse) {}
```